Makefile.am

```diff
@@ -0,0 +1,91 @@
+# $Id: Makefile.am,v 1.0.0-0 2016/05/06 14:24:14 Cnangel Exp $
```

Comment:
This introduces a third make system besides the existing Makefile and CMake. Plus this is multilevel. This may make it more cumbersome and error prone to maintain.